spk/minio/Makefile

```diff
@@ -1,10 +1,10 @@
 SPK_NAME = minio
-SPK_VERS = 2020-11-25T22-36-25Z
+SPK_VERS = 2021.06.14-012923
```

Comment:
SPK_VERS must not contain a `-` sign as only one is allowed.
The full version will bei `$(SPK_VERS)-$(SPK_REV)` (and SPK_REV is the only version key used by the spk repository). 
As we can ensure not to publish two versions at the same day, we could use `SPK_VERS = 2021.06.14` (and hopefully minio does not create two official releases on the same day too).

Reply:
Minio already did multiple releases the same day in the past. I guess this shouldn't be a big problem for us because the real version is still available in the cross makefile.

Fixed in https://github.com/SynoCommunity/spksrc/pull/4683/commits/7651bb27af4126305aaca5962b1aa1c15e4a042b